<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cSceneWindow</class>
 <widget class="QWidget" name="cSceneWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,1,0,0,5">
     <item row="3" column="0" colspan="8">
      <widget class="QTabWidget" name="m_lpTabs">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="m_lpTabDescription">
        <attribute name="title">
         <string>Description</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTextEdit" name="m_lpDescription"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="m_lpTabCharacter">
        <attribute name="title">
         <string>Character</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QTreeView" name="m_lpCharacterList">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="m_lpTabPlace">
        <attribute name="title">
         <string>Place</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QTreeView" name="m_lpPlaceList">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="m_lpTabObject">
        <attribute name="title">
         <string>Object</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QTreeView" name="m_lpObjectList">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QDateTimeEdit" name="m_lpFinishedAt">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Chapter:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="m_lpFinishedLabel">
       <property name="text">
        <string>Finished:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="7">
      <widget class="QLineEdit" name="m_lpName"/>
     </item>
     <item row="6" column="0" colspan="8">
      <widget class="cTextEdit" name="m_lpText"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>State:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Target Date:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="m_lpState"/>
     </item>
     <item row="5" column="3">
      <widget class="QDateTimeEdit" name="m_lpStartedAt">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="m_lpStartedLabel">
       <property name="text">
        <string>Started:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="QDateTimeEdit" name="m_lpTargetDate"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Part:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="7">
      <widget class="QLineEdit" name="m_lpPart">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="7">
      <widget class="QLineEdit" name="m_lpChapter">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ctextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_lpName</tabstop>
  <tabstop>m_lpText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
